heroku/github-triage
# ember install ember-cli-dotenv

## Why the change?

Fixes heroku/github-triage-squad#86

## How do I verify the change?



1. Pull down this branch
2. Create a .env file in your directory
3. Add a variable to it called `VARIABLE_NAME`
4. Try accessing that variable in the application using `process.env.VARIABLE_NAME` and verify that it works
